### DISCUSSION TITLE
why is Connection refused when I add  --web.enable-lifecycle command?

### DISCUSSION THREAD

**Discussion by BeIFreedom** [prometheus/prometheus#10573] [Q&A]
why is Connection refused when I add  --web.enable-lifecycle command?
prometheus:
      image: quay.io/prometheus/prometheus:latest
      container_name: prometheus
      network_mode: host
      restart: unless-stopped
      volumes:
        - /home/xxx/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
      command:
        - '--web.enable-lifecycle'
curl: (7) Failed to connect to ip port 9090: Connection refused

**Comment by roidelapluie** [comment]
There is no reason at the prometheus level. probably your container is not running.

**Comment by BeIFreedom** [comment]
The container running .And after I remove this command, I can access it


---Original---
From: "Julien ***@***.***&gt;
Date: Mon, Apr 11, 2022 21:59 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [prometheus/prometheus] why is Connection refused when I add --web.enable-lifecycle command? (Discussion #10573)




 
There is no reason at the prometheus level. probably your container is not running.

**Reply by BeIFreedom** [reply]
okI see what the reason is
